index.html

```diff
@@ -0,0 +1,526 @@
+
+            <div class="diagram-details">
+              <div class="diagram-detail">
+                <!--transparent borders-->
+                <div class="icon -borders"></div>
+                <!--background-color-->
+                <div class="icon -color">
+                  <!--img-->
+                  <img  class="resize" src="./public/img/icon-processing.png">
+                </div>
+                <p class="text -small -color-accent -caps -bold centered">Natural Language Processing</p>
+                <p class="text -small content">Florence’s conversational style will encourage users to speak to her like a friend. Natural language processing, like those offered by IBM’s Watson, will be needed to extract meaning and respond intelligently.</p>
+              </div>
+            </div>
+
+            <div class="diagram-details reverse">
+              <div class="diagram-detail">
+                <!--transparent borders-->
+                <div class="icon -borders"></div>
+                <!--background-color-->
+                <div class="icon -color">
+                  <!--img-->
+                  <img class="resize" src="./public/img/icon-behavioral.png">
+                </div>
+                <p class="text -small -color-accent -caps -bold centered">BEHAVIORAL MODELING</p>
+                <p class="text -small content">Florence can generate many different messages which, unchecked, could make her a little too friendly. A system that learns the preferences of users will be helpful in tailoring which kinds of messages are sent, when, and in what frequency.</p>
+              </div>
+            </div>
+
+            <div class="diagram-details reverse">
+              <div class="diagram-detail">
+                <!--transparent borders-->
+                <div class="icon -borders"></div>
+                <!--background-color-->
+                <div class="icon -color">
+                  <!--img-->
+                  <img class="resize" src="./public/img/icon-integration.png">
+                </div>
+                <p class="text -small -color-accent -caps -bold centered">IOT INTEGRATIONS</p>
+                <p class="text -small content">The emergence of connected adherence devices like Proteus pills and Smrxt bottles, combined with the popularity of health devices like Fitbits and Qardio, can provide a more rounded view of patients. Aggregating and synthesizing that information, perhaps using services like Validic, would help Florence interact more appropriately.</p>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+    </div>
+  <!--4) GREEN BACKGROUND END-->
+
+
+  <!--5) BLUE BACKGROUND START-->
+    <div class="bg-blue">
+      <!--FOOTER SECTION-->
+      <div class="content-wrap">
+        <footer>
+          <div class="centered viget">
+            <p class="text -x-small -color-white -caps -wide-spacing -bold">An Exploration by</p>
+            <div></div>
```

Comment:
If this is for holding the Viget logo, it should either have text in it that's visually hidden for screen readers or use an inline image with `alt` text. Either way, a class would be helpful here so there's an understanding of what this does.